ppr-ui/src/views/newMhrRegistration/MhrRegistration.vue

```diff
@@ -141,6 +147,9 @@ export default defineComponent({
       registrationTypeUI: computed((): UIRegistrationTypes => {
         return getRegistrationType.value?.registrationTypeUI || ('' as UIRegistrationTypes)
       }),
+      isDraft: computed((): boolean => {
+        return getMhrDraftNumber.value
+      }),
```

Comment:
If you prefer, you can also use the `getMhrDraftNumber` directly in the template without `.value` since the templates already deconstruct the refs. 
Your call but additional computed property is not really required!